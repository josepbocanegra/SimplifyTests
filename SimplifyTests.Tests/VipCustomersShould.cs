using System;
using System.Collections.Generic;
using System.Text;
using ApprovalTests;
using ApprovalTests.Reporters;
using Xunit;

namespace SimplifyTests.Tests
{
    [UseReporter(typeof(DiffReporter))]
    public class VipCustomersShould
    {
        [Fact]
        public void ReturnCustomersFullNameInAlphabeticalOrder()
        {
            var vipCustomers = new VipCustomers();
            vipCustomers.Add(new Customer("Mandy", "Harris"));
            vipCustomers.Add(new Customer("John", "Smith"));
            vipCustomers.Add(new Customer("Albert", "Thomson"));

            var orderedVipCustomers = vipCustomers.InAlphabeticalOrder();

            Approvals.VerifyAll(orderedVipCustomers, "");
        }

    }
}
