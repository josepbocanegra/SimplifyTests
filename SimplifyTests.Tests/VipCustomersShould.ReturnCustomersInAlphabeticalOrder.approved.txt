Albert Thomson
John Smith
Mandy Harris
